frontend/src/containers/Addgroup.js

```diff
@@ -0,0 +1,392 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import LoginTopBar from '../containers/LoginTopBar';
+import Notification from '../containers/Notification';
+
+import CustomTextArea from '../components/CustomTextArea';
+import ImagePicker from '../components/ImagePicker';
+import Input from '../components/CustomTextArea';
+import PublicFooter from '../components/PublicFooter';
+import Select from '../components/Select';
+
+import appendGroupForm from '../actions/form/append_group';
+import createGroup from '../actions/groups/create';
+import notify from '../actions/notification/notify';
+import uploadImage from '../actions/images/upload';
+import validateSchema from '../actions/form/validate_schema';
+
+import groupSchema from '../joi_schemas/group';
+
+import i18n from '../lib/i18n';
+
+const memberRole = 'MEMBER';
+
+export class Addgroup extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    const {
+      appendGroupForm,
+      groupForm,
+      i18n
+    } = this.props;
+
+    const {
+      name,
+      slug,
+      currency,
+      website,
+      video,
+      mission,
+      description,
+      longDescription,
+      whyJoin,
+      tags,
+      users1,
+      users2,
+      users3,
+      users4,
+      users5
+    } = groupForm.attributes;
+
+    return (
+      <div className='Login'>
+        <Notification {...this.props} />
+        <LoginTopBar />
+        <div className='Addgroup'>
+          <div className='Addgroup-header'>
+            <strong>Create a new collective</strong>
+          </div>
+            <div className="Addgroup-form-container">
```

Comment:
Double Quote `"` or Single Quote `'`   

I Believe single is the best choice, since it doest require a SHIFT key, and visually it looks slimmer/less.